activesupport/lib/active_support/dependencies/zeitwerk_integration.rb

```diff
@@ -59,7 +59,10 @@ def setup_autoloaders
               end
             end
 
-            Rails.autoloaders.each(&:setup)
+            Rails.autoloaders.each do |loader|
+              loader.inflector = ActiveSupport::Inflector
```

Comment:
What Inflector does the constantize and safe_constantize methods end up referring to? Zeitwerk’s or Active Support’s? I’d think the latter, so I’m confused why this is needed.

Also sounds like Zeitwerk should have a default_inflector akin to its logger.

Reply:
Correct, in this file `Inflector` is referring to the ActiveSupport `Inflector`, and everything in this file is all well and good. This setting is so that https://github.com/fxn/zeitwerk/blob/ccd4b76647e9ca90f0811b7b0f98782b07c4739a/lib/zeitwerk/loader.rb#L341 uses the same inflections this module would use to populate the constants that zeitwork then autoloads. In the test I added, if we didn't make this change, `zeitwerk` would populate `::GraphqlController` instead of `::GraphQLController`.